Makefile

```diff
@@ -19,7 +19,7 @@ test: ## Run tests
 	ruff check .
 	ruff format --check .
 	pytest -n auto
-	python setup.py sdist
+	python -m build
```

Comment:
got me thinking - what does this do? (i mean, it packages up the library in to a tar.gz in the dist folder

but do we need this? we don't have any ambitions to distribute this more widely where we'd need to build the package to push to pypi or something, and when we install `pip install notifications-utils @ git+https://<url>.git@1.0.0` that just creates a checkout

can we get rid of this completely?

Reply:
It helps detect mistakes which would make the package uninstallable.

For example if you remove the `name` line from `pyproject.toml`:
```diff
- name = "notifications-utils"
```

The tests and linters will still pass. But trying to run `pip install -e ../notifications-utils` from one of the apps will result in an error.

Running `python -m build` as part of the tests catches this error before it gets merged – you would see this error:
```
configuration error: `project` must contain ['name'] properties
```

I’m sure there’s another example which is a more likely mistake someone could make, but I can’t think of it right now.